app/views/devise/registrations/_admins.html.erb

```diff
@@ -0,0 +1,10 @@
+<% @admins.each do |admin| %>
+  <tr>
+    <td><%= admin.firstname %> <%= admin.lastname %>

```

Comment:
```suggestion
    <td><%= admin.firstname %> <%= admin.lastname %> </td>
```
Close all of the td tags on this to be consistent within this page as well as the site.